models/rentals.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+
+var sqlite3 = require('sqlite3').verbose();
+
+function Rental() {
+  this.tableName = "rentals";
+}
+
+Rental.prototype = require('./database').prototype;
+
+Rental.prototype.update = function update(movie_title, date, callback) {
```

Comment:
In retrospect, it seems like maybe this would be better named something like Rental#return, since this isn't a general update function but instead specifically for the return_date column?